logp/core.go

```diff
@@ -236,6 +236,22 @@ func makeEventLogOutput(cfg Config, enab zapcore.LevelEnabler) (zapcore.Core, er
 	return wrappedCore(core), nil
 }
 
+// WithFileOutput creates a new file output based on cfg and
+// replaces the previous one.
+func WithFileOutput(cfg Config) func(zapcore.Core) zapcore.Core {
+	out, err := makeFileOutput(cfg, zap.DebugLevel)
```

Comment:
Is this hard coding the debug level?

Reply:
It was, thanks for catching that. It's fixed on 5cacb6c

Reply:
Is there a way to update the test to have caught this?

Reply:
I believe be best approach is to test this function. It's a small one, so it should be simple